lib/Module/CMakeLists.txt

```diff
@@ -23,9 +23,14 @@ set(LLVM_COMPONENTS
   bitreader
   bitwriter
   codegen
+  instcombine
```

Comment:
Something is wrong here. Both the `llvm-config` and CMake approach should see that `transformutils` is a dependency of instcombine.

If you take a look at the installed `LLVMExports.cmake` file you'll see something like

```cmake
# Create imported target LLVMInstCombine
add_library(LLVMInstCombine STATIC IMPORTED)

set_target_properties(LLVMInstCombine PROPERTIES
  INTERFACE_LINK_LIBRARIES "LLVMAnalysis;LLVMCore;LLVMSupport;LLVMTransformUtils"
)
```